spec/rspec/expectations/failure_aggregator_spec.rb

```diff
@@ -262,10 +262,13 @@ def expect_error_included_in_aggregated_failure(error)
           end
         }.to fail_including { dedent <<-EOS }
           |  1) expected `1.even?` to return true, got false
+          |     ./spec/rspec/expectations/failure_aggregator_spec.rb:259:in `block (5 levels) in <module:Expectations>'
```

Comment:
Typically we'd interpolate the end of these lines, e.g:
```suggestion
          |     ./spec/rspec/expectations/failure_aggregator_spec.rb:#{__LINE__  - 6}:in `block (5 levels) in <module:Expectations>'
```
So that changes to the files don't break the tests.